pyanaconda/core/constants.py

```diff
@@ -135,9 +134,13 @@
 # - GUI starts this count once it finishes its initialization
 # - the geoloc thread is started early and in most cases will be already done
 #   when GUI finishes its initialization, so no delays will be introduced
-GEOLOC_TIMEOUT = 3
+GEOLOC_GUI_TIMEOUT = 3
```

Comment:
Shouldn't this timeout disappear?

Reply:
Probably yes. I just realized keeping it "in case it's needed again" is a bad idea.

If I change the waiting for result to using a signal, as suggested somewhere around, that actually removes the need for the other timeout, too.